stubs/frontend/exportPage/jsons/exportData.json

```diff
@@ -1,67 +1,67 @@
 {
   "velocity": {
-    "velocityForSP": "20",
-    "velocityForCards": "14"
+    "velocityForSP": 20,
+    "velocityForCards": 14
   },
   "cycleTime": {
-    "averageCircleTimePerCard": "30.26",
-    "averageCycleTimePerSP": "21.18",
+    "averageCycleTimePerCard": 30.26,
+    "averageCycleTimePerSP": 21.18,
     "totalTimeForCards": 423.59,
     "swimlaneList": [
       {
         "optionalItemName": "Analysis",
-        "averageTimeForSP": "8.36",
-        "averageTimeForCards": "11.95",
-        "totalTime": "167.27"
+        "averageTimeForSP": 8.36,
+        "averageTimeForCards": 11.95,
+        "totalTime": 167.27
       },
       {
         "optionalItemName": "In Dev",
-        "averageTimeForSP": "12.13",
-        "averageTimeForCards": "17.32",
-        "totalTime": "242.51"
+        "averageTimeForSP": 12.13,
+        "averageTimeForCards": 17.32,
+        "totalTime": 242.51
       },
       {
         "optionalItemName": "Waiting for testing",
-        "averageTimeForSP": "0.16",
-        "averageTimeForCards": "0.23",
-        "totalTime": "3.21"
+        "averageTimeForSP": 0.16,
+        "averageTimeForCards": 0.23,
+        "totalTime": 3.21
       },
       {
         "optionalItemName": "Done",
-        "averageTimeForSP": "86.20",
-        "averageTimeForCards": "123.14",
-        "totalTime": "1723.99"
+        "averageTimeForSP": 86.2,
+        "averageTimeForCards": 123.14,
+        "totalTime": 1723.99
       },
       {
         "optionalItemName": "Block",
-        "averageTimeForSP": "8.54",
-        "averageTimeForCards": "12.20",
-        "totalTime": "170.80"
+        "averageTimeForSP": 8.54,
+        "averageTimeForCards": 12.2,
+        "totalTime": 170.8
```

Comment:
Double Confirm: should we keep 2 dig like 170.80

Reply:
If you want to save the trailing 0 of a number in the JSON file, you must save the current value as a string. Maybe we can process saving the data to two decimal places when the page is displayed

Reply:
If keep 2 dig, need to use `toFixed(2)`.But this method returned value is string.
Step is:
170.8 -> 170.8.toFixed(2) -> parseFloat(number-> string -> number)
Is that expected?